extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties

```diff
@@ -29,6 +29,8 @@ thing-type.config.sonos.zoneplayer.udn.label = Eindeutiger Name (UDN)
 thing-type.config.sonos.zoneplayer.udn.description = Der UDN zur Identifizierung des Sonos Ger�tes.
 thing-type.config.sonos.zoneplayer.notificationTimeout.label = Timeout f�r Benachrichtigungen
 thing-type.config.sonos.zoneplayer.notificationTimeout.description = Timeout f�r Benachrichtigungen (in Sekunden).
+thing-type.config.sonos.zoneplayer.notificationVolume.label = Lautst�rke der Benachrichtigung
+thing-type.config.sonos.zoneplayer.notificationVolume.description = Zeigt die Lautst�rke f�r Benachrichtigungen (in %) an.
```

Comment:
Shouldn't it be "Die Lautstärke für ...". The refresh description uses "Intervall zur Abfrage" and not "Zeigt das ..."